src/components/GameText.vue

```diff
@@ -83,7 +81,7 @@ export default {
       config:{
         gameDone:false
       },
-      inputPlaceholder:"Start typing to initiate the test...",
+      inputPlaceholder:"Start typing the text here to initiate the Test ...",
```

Comment:
@zapobyte shouldn't be 'Start typing the text here to initiate the `game`' ?

Reply:
@oanaOM thanks for the suggestion. I've change the wording in the new release 👍🏻